pkg/collector/corechecks/network-devices/cisco-sdwan/payload/interfaces.go

```diff
@@ -0,0 +1,78 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+package payload
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/network-devices/cisco-sdwan/client"
+	devicemetadata "github.com/DataDog/datadog-agent/pkg/networkdevice/metadata"
+)
+
+// CiscoInterface is an interface to abstract the underlying network interface object (vEdge vs cEdge)
+type CiscoInterface interface {
+	// ID returns an unique interface ID
+	ID() string
+	// Index returns the interface index
+	Index() int
+	// Speed returns the interface speed
+	Speed() int
```

Comment:
Can the function name make explicit the unit we expect?

Reply:
Sure good call! Renamed in 36f38dd